src/data/battle_frontier/trainer_hill.h

```diff
@@ -2564,7 +2723,7 @@ static const struct TrHillFloor sDataTagVariety_Floors[] = {
                                             .spAttackIV = 30,
                                             .spDefenseIV = 31,
                                             .abilityNum = 0,
-                                            .personality = 0x202,
+                                            .personality = 0x0202,
```

Comment:
Why the extraneous zeros added to personality? It's a 32 bit number, so this wouldn't be the "full" number regardless

Reply:
I think the find/replace for the collision data accidentally caught this. I'll revert them.